mlflow/server/js/src/test-utils/ReduxStoreFixtures.js

```diff
@@ -31,6 +32,20 @@ export const addExperimentToState = (state, experiment) => {
   };
 };
 
+export const addExperimentTagsToState = (state, experiment_id, tags) => {
+  const oldExperimentTags = state.entities.experimentTagsByExperimentId;
+  return {
+    ...state,
+    entities: {
+      ...state.entities,
+      experimentTagsByExperimentId: {
+        ...oldExperimentTags,
+        experiment_id: tags,
```

Comment:
What if `experiment_id` has existing tags? This will flush the old tags under that id.

Reply:
Ah right - fixed that.